admin/data/migrations/migrations/20171019113308-edit-check-windows-status.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const config = require('../config')
+
+module.exports = {
+  up (db, next) {
+    mongoose.connect(config.mongodb.url, async error => {
```

Comment:
You don't really needed mongoose here as you are using a mongo-style update.

Reply:
We have a mix of mongoose and plain mongo in the migrations, I rather stay with this version of the script please, it works just fine.